Dockerfile

```diff
@@ -6,8 +6,6 @@ COPY startup.sh /startup.sh
 RUN apt-get update -y
 RUN apt-get upgrade -y
 RUN apt-get install -yq git gcc make build-essential python3-dev python3-reportlab
-RUN git clone -b 0.7.0 https://github.com/ONSdigital/sdx-common.git
-RUN pip3 install ./sdx-common
 
```

Comment:
Docker container isn't starting:
```
/usr/bin/python3: Error while finding spec for 'app.consumer' (<class 'ImportError'>: No module named 'sdx')
```

Looks like the dockerfile needs to import the `requirements.txt` and run a `pip3` install on it

Reply:
This has now been addressed and tested